src/sourceFiles/index.ts

```diff
@@ -956,4 +1012,24 @@ export namespace SourceFilesModel {
     export interface ListReviewedSourceFilesBuildOptions extends PaginationOptions {
         branchId?: number;
     }
+
+    export interface User {
```

Comment:
I believe there should be an existing user Interface

Reply:
It looks like we should follow the current pattern of having separate models, such as issues, stringComments, and stringTranslations have. I just checked the actual API call, and it returns this short user model.